include/crm/common/ipc.h

```diff
@@ -135,6 +135,7 @@ enum crm_ipc_flags
 {
     crm_ipc_flags_none              = UINT32_C(0),
     //! Message has been compressed
+    // @COMPAT crm_ipc_compressed is unused and deprecated since 3.0.1
```

Comment:
Replace both of these lines with `//! \deprecated Since 3.0.1` or similar. Let's use the Doxygen directive for deprecation like we do in (for example) `compat.h` headers.